content/ADR-138-extended-passport-support.md

```diff
@@ -0,0 +1,73 @@
+---
+adr: 138
+date: 2022-11-25
+title: Technical Assessment for the Extended Passport
+authors:
+  - manumena
+status: Draft
+type: RFC
+spdx-license: CC0-1.0
+redirect_from:
+  - /rfc/RFC-1
+---
+
+## Need
+
+A New Passport view is being developed by the Explorer.
+This view will require storing information in the profile entity and retrieving things as collectibles to be displayed, as well as being able to block, add as a friend or initiate a chat.
+
+## Requirements
+The applicable requirements from the Catalyst point of view:
+
+1. Intro of a user:
+Use the "description" field from profiles to use in the "Intro" section or create a new one if it is different from the "description".
+
+2. Block/Unblock users:
+Use the "blocked" field from profiles to see/report blocked users.
+
+3. Equipped wearables:
+Take the information from the profile. There are the equipped wearables, emotes and the current name.
+
+4. Items: wearables/emotes/names/lands:
+For the wearables and emotes there are two available endpoints `/wearables-by-owner` and `/emotes-by-owner` that can be used for that. It will be necessary to create `/names-by-owner` and `/lands-by-owner` endpoints with similar behavior.
```

Comment:
Let's try to define whether we'll be needing pagination or not. If we do, I'd probably create new endpoints for all, as the existing ones will require breaking changes. If no pagination is needed, then we're good to implement the newer ones just the same.

Reply:
👍🏻